.github/dependabot.yml

```diff
@@ -22,6 +22,10 @@ updates:
     directory: "common-gradle-enterprise-gradle-configuration"
```

Comment:
i think we should rename this one to `-groovy` in order to prevent confusion.

Reply:
done!